### ISSUE TITLE
Bring back swap interval when vsync = none is used with the glx backend

### ISSUE THREAD

**Issue by SoniEx2** [yshui/compton#136]
Previously, in 4.0, I didn't have tearing because vsync = none + glx had swap interval. Now, in 5.1, I do.

I want my vsync set to none so programs can actually render at whatever framerate they want (e.g. 200 FPS in Minecraft), but I want the results to be synced to the screen at the correct time.

**Comment by yshui** [comment]
@SoniEx2 
1. In 4.0, vsync _was enabled_ even when you set it to `none`. Now it isn't.
2. Whether compton has vsync enabled shoudn't affect what your programs do. (I could be wrong, though)

**Comment by SoniEx2** [comment]
hm, I'm not sure why that's working now. previously it'd force everything to render at 60 FPS, so I had to use vsync = none and backend = glx. this was fine until 5.1 changed it, but you're right that I'm no longer getting tearing with non-none vsync and it's not forcing everything to render at 60 FPS.

(I've been using compton since 2016 apparently, so my configs have gone through quite a lot. so I've no idea when that "force everything to vsync" issue was fixed.)
